src/Sulu/Component/Content/Metadata/Parser/PropertiesXmlParser.php

```diff
@@ -345,7 +345,12 @@ private function loadParam(\DOMXPath $xpath, \DOMNode $node)
                 $result['value'] = $this->loadParams('x:param', $xpath, $node);
                 break;
             default:
-                $result['value'] = $this->getValueFromXPath('@value', $xpath, $node, 'string');
+                $result['value'] = $this->getValueFromXPath('@value', $xpath, $node);
+
+                if (null === $result['value']) {
```

Comment:
will fix this in SingleSelect by using always name

Reply:
So this PR is not needed anymore? Can it be closed then?